Cargo.lock

```diff
@@ -6,8 +6,9 @@ dependencies = [
  "rand 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "regex 0.1.19 (registry+https://github.com/rust-lang/crates.io-index)",
  "regex_macros 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
- "rustc-serialize 0.3.1 (registry+https://github.com/rust-lang/crates.io-index)",
- "time 0.1.19 (git+https://github.com/rust-lang/time)",
+ "rustc-serialize 0.3.3 (registry+https://github.com/rust-lang/crates.io-index)",
+ "strenum 0.0.3 (registry+https://github.com/rust-lang/crates.io-index)",
+ "time 0.1.20 (git+https://github.com/rust-lang/time)",
```

Comment:
Have you tried the new time crate from crates.io? Maybe your bug is already fixed.